### DISCUSSION TITLE
Installation issue - python

### DISCUSSION THREAD

**Discussion by SunilAnandatheertha** [BlueQuartzSoftware/DREAM3DNX-Issues#17] [Q&A] [answered]
Hi all. I am facing the below issue while trying to install in the local env. I am hoping to get some help on how to overcome this. Currently I am using miniconda3. Thank you.

(nxpython) C:\Users\......>conda install -c bluequartzsoftware dream3dnx
Channels:
 - bluequartzsoftware
 - defaults
Platform: win-64
Collecting package metadata (repodata.json): done
Solving environment: failed

LibMambaUnsatisfiableError: Encountered problems while solving:
  - nothing provides fmt 10.* needed by dream3dnx-0.2.00-py310hc809458_0

Could not solve for environment specs
The following package could not be installed
└─ dream3dnx is not installable because it requires
   └─ fmt 10.* , which does not exist (perhaps a missing channel).

**Comment by imikejackson** [comment, marked as answer]
@SunilAnandatheertha From this page: https://www.dream3d.io/python_docs/
```
conda config --add channels conda-forge
conda config --set channel_priority strict
conda create -n nxpython python=3.12
conda activate nxpython
conda install -c bluequartzsoftware dream3dnx
```
You are missing the conda-forge channel.
